string table_name
string direction
int32 duration
---
string reply
